contributing_to_docs/doc_guidelines.adoc

```diff
@@ -257,6 +257,29 @@ link:https://redhat-documentation.github.io/modular-docs/#creating-procedure-mod
 When needed, use `.Prerequisites`, `.Next steps`, or `.Additional resources` syntax to suppress TOC formatting within a module. Do not use `==` headings or xrefs in modules.
 ====
 
+[id="using-conscious-language"]
+== Using conscious language
+
+To assist with the removal of problematic language from the documentation, use the following terminology when referring to OpenShift control plane or compute nodes:
+
+[options="header"]
+|===
+|Branch |Control plane node reference |Compute node reference
```

Comment:
This also does apply to master references that aren't nodes. In case you want to work that in somehow. For example, there are plenty of instances of "master" without node afterward, and "node" isn't always the right thing to add. It could be "host" or "machine", or possibly something else.

Reply:
Thank you for highlighting this. I have added a note under the table stating:

"You can replace "node" in the preceding examples with "machine", "host", or another suitable description."